src/pages/guide/screens/Guide/Guide.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { PageReturn } from '@/components';
+import { BUY_SELL_URL } from '@/constants';
+import { Localize, useTranslations } from '@deriv-com/translations';
+import { Text, useDevice } from '@deriv-com/ui';
+import { LocalStorageUtils } from '@deriv-com/utils';
+import { Awareness } from '../Awareness';
+import { Blog } from '../Blog';
+import { FAQs } from '../FAQs';
+import { GettingStarted } from '../GettingStarted';
+import { Videos } from '../Videos';
+import './Guide.scss';
+
+const Guide = () => {
+    const history = useHistory();
+    const { localize } = useTranslations();
+    const { isDesktop } = useDevice();
+
+    useEffect(() => {
+        LocalStorageUtils.setValue<boolean>('should_show_p2p_guide', false);
+    }, []);
```

Comment:
wait shouldn't we allow the users to come back and visit the guide whenever they want to, or is this how it behaves